crates/router/src/compatibility/stripe/payment_intents/types.rs

```diff
@@ -266,9 +344,9 @@ impl TryFrom<StripePaymentIntentRequest> for payments::PaymentsRequest {
             mandate_data: mandate_options,
```

Comment:
Hey so, what are we doing with the `item.mandate_data`, that is present in the request

Reply:
Not doing anything as of now since mapping was already there. But it turns out that the mapping was only for a specific use-case. So, I'll add this mapping and remove that one.